### ISSUE TITLE
[nu-validator coverage] Two ARIA fixtures remain nu-only

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3999]
`tests/external/CLAUDE.md` states `nu-only = 0` as an invariant. Two fixtures currently violate it. Both are genuine coverage gaps: markuplint reports **0 violations** on each with current `dev` (verified by regenerating the markuplint side of the snapshot).

### `html-aria/misc/aria-hidden-true-with-hidden-until-found-novalid.html`

```html
<div hidden="until-found" aria-hidden="true">Hidden content</div>
```

> nu: Attribute “aria-hidden” with value “true” must not be specified on elements with “hidden” attribute value “until-found”. This combination prevents content from being accessible to assistive technology when revealed through search.

### `html-aria/presentational-children/testcase-844.html`

`<div role="math">` containing `label`, `input`, `a[href]`, and `role="button"`.

> nu: An element with the attribute “role=button” must not appear as a descendant of an element with the attribute “role=math”. (plus four more of the same shape)

## Resolution

Per the `bench-triage` skill, each fixture needs to reach one of two verdicts: implement the check in markuplint so it becomes `match-error`, or record a spec-cited entry in `excluded-ids.json` so it becomes `nu-over`.

## Prerequisites

- **Docker is required** to regenerate the nu-validator side.
- The scraper fix for the MDN Baseline badge rename changes markuplint's output, so the order is: land that fix → regenerate snapshots → triage.
- The committed snapshot predates the merge of #3989 (generated 2026-08-26 16:28 JST, merged 17:31 JST), so it needs regenerating regardless.

## Notes for whoever picks this up

Three things make the benchmark hard to reproduce locally; none are blockers but all cost time if hit cold:

1. **Local nu raw snapshots can be built from a different validator image than the one `meta.json` records** (observed: all local snapshots on one digest, `meta.json` on another). Nothing detects the mismatch, so a comparison can silently mix validator versions.
2. **Raw snapshots are gitignored**, so they go stale without any signal (observed: 5441 of 5442 files months old).
3. **`yarn bench:update:ml --filter <one file>` regenerates that one file and then unconditionally rewrites the full aggregate** — `meta.json`, `diff/*`, `summary.md`, and the generated spec — from whatever raw snapshots are on disk (`tests/external/bench/update-snapshots.ts:102-129`). Mixing one fresh file with stale ones produced a plausible-looking but wrong `nu-only` count. `--skip-refresh` exists but is not applied automatically for filtered runs.
